components/brave_ads/browser/ads_service.cc

```diff
@@ -51,6 +51,12 @@ void AdsService::RegisterProfilePrefs(
   registry->RegisterBooleanPref(
       ads::prefs::kShouldAllowConversionTracking, true);
 
+  // Diagnostics
+  registry->RegisterIntegerPref(ads::prefs::kEligibleAdsCount, -1);
```

Comment:
Why are we persisting some values in prefs as all of these values can be returned in the serialized JSON from the ads library?